glusterd2/bricksplanner/subvoltype_replicate.go

```diff
@@ -44,3 +46,18 @@ func (s *replicaSubvolPlanner) BrickType(idx int) string {
 func init() {
 	subvolPlanners["replicate"] = &replicaSubvolPlanner{}
 }
+
+// ArbiterBrickSizeCalculation returns the arbiter brick size for that particular volume
+func ArbiterBrickSizeCalculation(brickSize uint64, avgFileSize uint64) (uint64, error) {
+	if avgFileSize == 0 {
+		return 0, errs.New("average file size is passed as zero")
+	}
+	// All the calculations should be in MB
+	arbiterSize := (4 / 1024) * (brickSize / avgFileSize)
+	// Assigning arbiter brick size to be bricksize if its lesser than 20M
+	// TODO: the size is calculated in MB, should be changed as per the default.
+	if arbiterSize < 20 {
```

Comment:
Do not hard code 20 here. Use the constant defined. I feel we should increase the minimum volume size to `100MB`